src/components/Menu/Menu.js

```diff
@@ -87,20 +93,18 @@ export function Menu(props) {
                 id="dropdownNavbar"
                 className="sm:ds-absolute sm:ds-w-[260px] dropdownShadow ds-text-[#284162] ds-bg-white"
                 aria-labelledby="dropdownLargeButton"
-                ref={dropdown}
               >
                 {menuList.map((element, index) => {
                   const Component = element?.component || "a";
-                  return (
-                    <Component
-                      href={element.path}
-                      className={`${
-                        index === 0 ? "ds-border-none" : "ds-border-t-2"
-                      } ds-font-body ds-flex ds-items-center ds-h-[55px] ds-px-4 hover:ds-text-[#0535D2] focus:ds-outline-none ds-ring-offset-2 focus:ds-ring-2 ds-ring-[#0535D2] ds-rounded-sm  focus:ds-border-none`}
-                      key={element.key}
-                      onClick={onClick}
-                    >
-                      <span className="ds-flex ds-items-center">
+                  return Component !== "a" ? (
+                    <Component href={element.path}>
+                      <a
```

Comment:
Would this throw any kind of accessibility or semantic warning? I'm thinking that `Component` can be a next `Link`, which renders to an `<a/>` right?

Reply:
I changed it to <a> because <span> doesn't work as a child element in Link unless you use the `legacyBehaviour` and `passHref` props for next/link. There is an eslint warning in the file complaining the <a> tag doesn't have `href`, because if user is using next/link, the `href`  prop needs be in <Link>